UNRELEASED.md

```diff
@@ -7,6 +7,7 @@
 - Added `external` prop to `ResourceList` ([#2408](https://github.com/Shopify/polaris-react/pull/2408))
 - Added `onMouseEnter` and `onTouchStart` props to `Button` ([#2409](https://github.com/Shopify/polaris-react/pull/2409))
 - Added `ariaHaspopup` prop to `Popover` ([#2248](https://github.com/Shopify/polaris-react/pull/2248))
+- Migrated `Dropzone` to a functional component and reduce complexity ([#2360](https://github.com/Shopify/polaris-react/pull/2360))
```

Comment:
We usually put these under ` Code quality`

Reply:
However, in this situation, it's not just a migration. I've changed the business logic. Even so, do you think it should still be in `code quality`? I could lean towards `Enhancements ` or `Enhancements `, thoughts?

Reply:
It works here, the key being "and reduce complexity".